### PR TITLE
Quizzes formik validation

### PR DESCRIPTION
**Pull request by TirlaP**

### CODE REVIEW

File: src/features/create-quiz/create-quiz-second-card/components/answer-item.tsx [hunk 1]
```diff
@@ -20,27 +24,41 @@ export const AnswerItem: React.FC<AnswerItemProps> = ({
...
   handleChange,
+  getError,
+  formValidation,
+  handleRadioChange,
 }) => {
   return (
     <div className="create-quiz__answer">
-      <div className="create-quiz__input-label">Answer {`0${index + 1}`}</div>
+      <label htmlFor="question" className="create-quiz__input-label">
+        Answer {`0${index + 1}`}*
```
Comment (madalina-agh) [inline src/features/create-quiz/create-quiz-second-card/components/answer-item.tsx]: Is this displayed well if there are more than 9 answers?

File: src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx [hunk 2]
```diff
@@ -1,182 +1,289 @@
...
     newList.push({ isCorrectAnswer: false, answerName: "" });
+    formik.values.answerList = [...newList];
     setAnswerList([...newList]);
   };
 
   const removeAnswer = (index: number) => {
-    let newList = answerList;
-    newList.splice(index, 1);
-    console.log(`Deleted item ${index}`);
+    let newList = answerList.filter((_, answerIndex) => answerIndex !== index);
```
Comment (madalina-agh) [inline src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx]: To remove an item from an array by its index you could also use the Javascript function `splice()`.

File: src/features/create-quiz/store/CreateQuizStore.tsx [hunk 3]
```diff
@@ -3,50 +3,63 @@ import { Question } from "../create-quiz-second-card/create-quiz-second-card";
...
   questions: QuestionItem[] = [];
-  selectedQuestion: SelectedQuestion | null = null;
+  selectedQuestionID?: string;
 
   constructor() {
     makeAutoObservable(this, {}, { autoBind: true });
   }
 
-  addQuestion(question: Question | null) {
+  addQuestion(question: Question | null, isCorrectResponse: number | null) {
```
Comment (madalina-agh) [inline src/features/create-quiz/store/CreateQuizStore.tsx]: The parameter `isCorrectResponse` seems to be a boolean from its notation, but I see it is not. I suggest changing it to something like `correctResponseIndex`.

**Review by madalina-agh — APPROVED** [review]

File: src/features/create-quiz/create-quiz-second-card/components/answer-item.tsx (hunk 1 above)
Comment (TirlaP) [reply src/features/create-quiz/create-quiz-second-card/components/answer-item.tsx]: I changed it now, and it will be displayed well.

**Pull request merged**